docs/painless/painless-operators-general.asciidoc

```diff
@@ -0,0 +1,426 @@
+[[painless-operators-general]]
+=== Operators: General
+
+[[precedence-operator]]
+==== Precedence
+
+An expression encapsulated within a *precedence operator* overrides
+existing precedence relationships between operators and is evaluated
```

Comment:
Need to call out the use of parens. The use of "precedence operator" seems a bit awkward in this case--and doesn't need to be highlighted, especially on the second occurrence. I might be inclined to go with something like, "An expression encapsulated by the precedence operator (enclosed in parentheses)..."

Reply:
Cleaned up.  I removed the format sentence from each operator given the grammar and examples.  Is this something I should add back in for each operator?

Reply:
Also removed the bolding of the operators from all of the general section.